posthog/models/person/util.py

```diff
@@ -31,6 +31,12 @@
 from posthog.models.utils import UUIDT
 from posthog.settings import TEST
 
+# The placeholder value that is written to the ClickHouse copy of the
+# ``PersonDistinctId`` data set after person deletion has occurred.
+# TODO: It would be nice to replace this with ``None`` (``NULL`` in ClickHouse)
+# in the future for clarity and consistency with the Postgres side.
+DELETED_PERSON_UUID_PLACEHOLDER = UUID(int=0)
```

Comment:
Clickhouse has a `isZeroOrNull` function we can use to check for both in the meantime. 

I'm also trying to think if using the default zero value of UUIDs on PG will have any impact on Clickhouse, but can't think of anything.